presto-product-tests-launcher/config/launcher-logging.properties

```diff
@@ -0,0 +1,3 @@
+org.testcontainers=DEBUG
+com.github.dockerjava=DEBUG
+io.prestosql=DEBUG
```

Comment:
Isn't that too much? How big are the log files?

Reply:
I've dropped that commit